### DISCUSSION TITLE
Unfriendly documents for newcomer

### DISCUSSION THREAD

**Discussion by bob-dw** [apache/dubbo#13836] [General]
这新手文档对新手也未免太不友好了，你确定一步一步的执行就能得到文档编辑者想要的结果？

**Comment by fengluo97** [comment]
除了官方文档，也可以结合其他手段入门，学有所成之后，再来一起优化文档~

**Comment by dejavuwl** [comment]
起了个zookeeper，然而继续起service-provider的时候根本没有配置registry，启动成功也并没有注册到zookeeper，这是没人维护吗

**Reply by dejavuwl** [reply]
https://dubbo.apache.org/en/docs3-v2/java-sdk/quick-start/brief/ 看起来这个URL对应的是3.2，也许这个地址的文档可能已经过期了

**Reply by dejavuwl** [reply]
至少应该显示一下当前文档对应的代码版本或者更新日期
